obp_private_alb_config/private-alb-waf.tf

```diff
@@ -1,3 +1,9 @@
+resource "aws_wafv2_ip_set" "internal_ips" {
+  name               = "internal_IPs"
+  scope              = "REGIONAL"
+  ip_address_version = "IPV4"
+  addresses          = ["10.0.0.0/16"]
```

Comment:
you can use the variable vpc_cidr_block

Reply:
ah no, that's in the other repo (derp)

well anyways, this could be a variable, but I don't oppose to merge as-is for now :)